File: core/src/com/mygdx/game/GdxGame.java
package com.mygdx.game;

import com.badlogic.gdx.Game;
import com.mygdx.game.screens.GameScreen;

public class GdxGame extends Game {

	@Override
	public void create () {
		setScreen(new GameScreen());
	}

	@Override
	public void render () {
		super.render();
	}
	
	@Override
	public void dispose () {
	}
}

File: core/src/com/mygdx/game/GdxGame.kt
package com.mygdx.game

import com.badlogic.gdx.Game
import com.mygdx.game.api.GameSettings
import com.mygdx.game.managers.ScreenManager
import com.mygdx.game.managers.ScreenManager.Screens.*

class GdxGame : Game() {

    override fun create() {
        ScreenManager.game = this
        ScreenManager.setScreen(GAME_SCREEN)
        GameSettings.DEBUG.state = true
    }

    override fun render() {
        super.render()
    }

    override fun dispose() {}
}
